src/components/DatePickerDocked/utils.ts

```diff
@@ -1,7 +1,20 @@
-import type { TextStyle, ViewStyle } from "react-native";
-import type { ComponentStylePropWithVariants } from "src/types";
+import type { TextStyle, ViewStyle } from 'react-native';
+import type { ComponentStylePropWithVariants } from 'src/types';
 
-export const MONTHS_DATA = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"]
+export const MONTHS_DATA = [
```

Comment:
The array is not required. make use of DateFns to get display month names

Reply:
I removed it from Month Component but it will still be used to render all the months in the MonthPicker component, otherwise we will have to create dummy dates to get each month with date-fns. @shashank1010

Reply:
you'll create dummy dates only once on mount and they'll be discarded immediately too.